packages/components/src/components/Badge/index.tsx

```diff
@@ -4,7 +4,7 @@ import { Stack } from '../Stack';
 import { Text } from '../Text';
 
 export interface BadgeProps {
-  variant?: 'trial' | 'neutral' | 'warning' | 'highlight' | 'error';
+  variant?: 'trial' | 'neutral' | 'warning' | 'highlight';
```

Comment:
Should we keep this? The `Badge` in prism has an `error` variant.

Reply:
Oh I didn't realise! I created this variant in the previous PR where I also added the mid-trial modal. Next week Alex and I will take a quick look into using prism components in v1, and it would be great if we can replace the badge with that one :)